editor/editor_inspector.cpp

```diff
@@ -4088,6 +4167,55 @@ void EditorInspector::_node_removed(Node *p_node) {
 	}
 }
 
+void EditorInspector::_configuration_info_changed(Object *p_object) {
+	if (object == p_object) {
+		// Only update the tree if the list of configuration info has changed.
+		if (_update_configuration_info()) {
+			update_tree_pending = true;
+		}
+	}
+
+	for (int i = 0; i < get_child_count(); i++) {
+		Array args;
+		args.append(p_object);
+
+		Node *child = get_child(i);
+		child->propagate_call(StringName("_configuration_info_changed"), args);
+	}
+}
+
+bool EditorInspector::_update_configuration_info() {
+	bool changed = false;
+	LocalVector<int> found_indices;
+	const Vector<ConfigurationInfo> config_info_list = EditorConfigurationInfo::get_configuration_info(object);
+
+	// New and changed entries.
+	for (const ConfigurationInfo &config_info : config_info_list) {
+		if (config_info.get_property_name().is_empty()) {
```

Comment:
There is a problem here. This code was written with the assumption that the inspector only needs to be updated if a property has a name, since that's when the tiny ⚠️ next to its name can (dis)appear.

But now that the inspector plugin lists *all* info at the top, this logic must be updated to also take propertyless info into account.

Without this fix, the inspector banner at the top will miss some config info updates.

Reply:
OK it was just a matter of removing the property_name check, easy fix.
I also renamed the variable from `property_configuration_info` to `configuration_info_cache` accordingly since it now caches all config infos on the node/resource.